src/k8s_helpers.rs

```diff
@@ -49,3 +55,67 @@ pub fn create_selector(key: &str, value: &str) -> BTreeMap<String, String> {
     btree.insert(key.to_string(), value.to_string());
     btree
 }
+
+#[allow(clippy::too_many_arguments)]
+pub fn create_replica_set(
+    name: &ValidatorType,
+    namespace: &str,
+    label_selector: &BTreeMap<String, String>,
+    image_name: &DockerImage,
+    environment_variables: Vec<EnvVar>,
+    command: &[String],
```

Comment:
nit: since the refs here all end up getting cloned, you may as well take them by value instead of ref here, and leave the burden of copies on the caller